servers/omnisharp/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian:bullseye-slim
+
+WORKDIR /tmp
+
+RUN apt-get update \
+  && apt install curl -y \
```

Comment:
One more change please - gotta get all instances of it.

Reply:
Sorry - I should have noticed that.